src/status_im/utils/transactions.cljs

```diff
@@ -6,6 +6,7 @@
 (defn get-network-subdomain [network]
   (case network
     "testnet" "ropsten"
+    "testnet_rpc" "ropsten"
```

Comment:
What is difference in testnet vs testnet_rpc?

Reply:
second uses RPC proxy